File: HelpdeskSystem/Helpdesk.Common/DTOs/QueueItemDTO.cs
using System;

namespace Helpdesk.Common.DTOs
{
    /// <summary>
    /// Data transfer object used to represent a queue item
    /// </summary>
    public class QueueItemDTO
    {
        public int ItemId { get; set; }
        public int? CheckInId { get; set; }
        public int StudentId { get; set; }
        public string Nickname { get; set; }
        public int TopicId { get; set; }
        public string Topic { get; set; }
        public string Description { get; set; }
        public string Unit { get; set; }
        public DateTime TimeAdded { get; set; }
        public DateTime? TimeHelped { get; set; }
        public DateTime? TimeRemoved { get; set; }
    }
}

File: HelpdeskSystem/Helpdesk.Website/ClientApp/src/app/data/DTOs/queue-item.dto.ts
export class QueueItem {
    public itemId: number;
    public checkInId?: number;
    public nickname: string;
    public topic: string;
    public description: string;
    public unit: string;
    public TimeAdded: Date;
    public TimeHelped?: Date;
    public TimeRemoved?: Date;
}
